src/main/kotlin/view/OutputView.kt

```diff
@@ -18,6 +31,10 @@ class OutputView {
         println()
     }
 
+    fun displayTicketWithoutBrackets(ticket: model.Ticket) {
+        println(ticket.numbers.sorted().joinToString(", "))
+    }
+
```

Comment:
I see some places where we can apply [this comment.](https://github.com/herotechcourse/kotlin-lotto/pull/2#discussion_r2190373850)